base_export_async/models/delay_export.py

```diff
@@ -101,40 +103,30 @@ def export(self, params):
         )
         date_today = fields.Date.today()
         expiration_date = fields.Date.to_string(
-            date_today + relativedelta(days=+int(time_to_live))
+            date_today + relativedelta(days=int(time_to_live) + 1)
         )
 
-        # TODO : move to email template
         odoo_bot = self.sudo().env.ref("base.partner_root")
         email_from = odoo_bot.email
         model_description = self.env[model_name]._description
-        self.env["mail.mail"].create(
+
+        export_record.write(
             {
+                "url": url,
+                "expiration_date": expiration_date,
+                "model_description": model_description,
+            }
+        )
+
+        self.env.ref("base_export_async.delay_export_mail_template").send_mail(
+            export_record.id,
+            email_values={
                 "email_from": email_from,
                 "reply_to": email_from,
-                "email_to": user.email,
-                "subject": _("Export {} {}").format(
-                    model_description, fields.Date.to_string(fields.Date.today())
-                ),
-                "body_html": _(
-                    """
-                <p>Your export is available <a href="{}">here</a>.</p>
-                <p>It will be automatically deleted the {}.</p>
-                <p>&nbsp;</p>
-                <p><span style="color: #808080;">
-                This is an automated message please do not reply.
-                </span></p>
-                """
-                ).format(url, expiration_date),
-                "auto_delete": True,
-            }
+            },
```

Comment:
should be force_send=True?

Reply:
I am not sure why we need force_send here.

Reply:
[`:param bool force_send: send email immediately; otherwise use the mail queue (recommended);`](https://github.com/odoo/odoo/blob/14.0/addons/mail/models/mail_template.py#L247)